content/pt-br/docs/reference/using-api/health-checks.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Kubernetes API health endpoints
+reviewers:
+- logicalhan
+content_type: conceito
+weight: 50
+---
+
+<!-- overview -->
+O Kubernetes {{< glossary_tooltip term_id="kube-apiserver" text="servidor de API" >}} fornece endpoints de API para indicar o status atual do servidor de API. Esta página descreve esses endpoints da API e explica como você pode usá-los.
+
+<!-- body -->
+
+## Endpoints da API para health
+
+O Kubernetes servidor de API fornece 3 endpoints de API (`healthz`, `livez` and `readyz`) para indicar o status atual do servidor de API.
```

Comment:
This can be improved, like: `O servidor de API do kubernetes fornece 3 endpoints para...`